desc/compute/utils.py

```diff
@@ -646,6 +532,52 @@ def tupleset(t, i, value):
     return res
 
 
+def _get_grid_surface(grid, surface_label):
+    """Return grid quantities associated with the given surface label.
+
+    Parameters
+    ----------
+    grid : Grid
+        Collocation grid containing the nodes to evaluate at.
+    surface_label : str
+        The surface label of rho, theta, or zeta.
+
+    Returns
+    -------
+    unique_size : ndarray
```

Comment:
If changes are requested in this PR, then remind myself to change ndarray to int